src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -327,7 +328,7 @@ export default {
 @import '../../../../assets/variables';
 
 .message {
-	padding: 4px 0 4px 0;
+	padding: 12px 8px;
```

Comment:
this looks a bit too drastic for a lot of system messages in a row (e.g. after a bigger call, or when inviting a group)